### PR TITLE
Improve the directory structure.

### LINKED ISSUE #1: the project structure does not conform to the standard library structure
The project directory structure does not look good, compared to the standard library structure. 
for making packages for PyPI the directory structure should be something like this
```
project-root
├──src/
   └──package_name/
      └──__init__.py
├──tests/
   └──package_name/
├──.coveragerc
├──.gitignore
├──LICENSE
├──README.md
├──setup.cfg
└──setup.py
```

### PR DESCRIPTION
**Pull request by sk-ip**
references #1 closes #1 
made the directory structure according to standard python library structure.

### CODE REVIEW

**Comment by sk-ip** [comment]
Some of the fields in setup.py file needs to be edited by the maintainer.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import setuptools
+
+setuptools.setup(
+    name="udiff",
+    version="",
```
Comment (hameerabbasi) [inline setup.py]: This would be version `0.6.0`

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    author="authors name",
```
Comment (hameerabbasi) [inline setup.py]: `Hameer Abbasi`

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    author_email="you@yourdomain.com",
```
Comment (hameerabbasi) [inline setup.py]: `hameerabbasi@yahoo.com`

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        "": "src"
+    },
+    include_package_data=True,
+    install_requires=(
+        "uarray", "unumpy",
+    ),
+    classifiers=[
+        "Development Status :: 1 - Alpha",
+        "Natural Language :: English",
+        "Intended Audience :: Developers",
```
Comment (hameerabbasi) [inline setup.py]: Add "Scientists" and "Mathematicians"

File: src/udiff/_diff_array.py [hunk 5]
```diff
@@ -7,7 +7,7 @@ def __init__(self, arr, diffs=None):
             self._diffs = arr.diffs
             self._arr = arr.arr
             self._var = arr.var
-        
+
```
Comment (hameerabbasi) [inline src/udiff/_diff_array.py]: Can you revert this line?

**Review by hameerabbasi — COMMENTED** [review]
This is a good start, thanks. Here are the missing pieces.

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    packages=setuptools.find_packages(
+        where="src",
+        exclude=["tests*"]
+    ),
+    package_dir={
+        "": "src"
+    },
+    include_package_data=True,
+    install_requires=(
+        "uarray", "unumpy",
```
Comment (hameerabbasi) [inline setup.py]: Need both to be `>= 0.6.0`.

**Review by hameerabbasi — APPROVED** [review]

**Comment by hameerabbasi** [comment]
Thanks @sk-ip!

**Pull request merged**